go.mod

```diff
@@ -177,6 +177,8 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+require github.com/frankban/quicktest v1.14.4 // indirect
```

Comment:
Please move this to the above `require` group

Reply:
done in 
4084690